### PR TITLE
Update icims-tutorial.md for universal login

### PR DESCRIPTION
**Pull request by kschmi-icims**
Updating the guide for setting up iCIMS with EntraID.

### CODE REVIEW

**Comment by kschmi-icims** [comment]
@microsoft-github-policy-service agree company="iCIMS"

File: docs/identity/saas-apps/icims-tutorial.md [hunk 1]
```diff
@@ -2,17 +2,17 @@
 title: 'Tutorial: Microsoft Entra integration with ICIMS'
 description: Learn how to configure single sign-on between Microsoft Entra ID and ICIMS.
 
-author: jeevansd
+author: kschmi-icims
```
Comment (Court72) [inline docs/identity/saas-apps/icims-tutorial.md:5]:
```suggestion
author: jeevansd
```

**Comment by Court72** [comment]
@jeevansd

Can you review the proposed changes? 
	
**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 
	
#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by bhavanabhavvs** [comment]
@kschmi-icims Thanks for the changes, we have now created a separate PR in our private repo for better validation. Please expect the changes to be live in 1-2 days. #please-close

**Pull request closed without merging**